File: PayJunction VB/PayJunctionLibrary/PayJunction/Response.vb
Namespace PayJunction
    MustInherit Class Response
        Public Property Errors() As List(Of ResponseError)
            Get
                Return m_Errors
            End Get
            Set
                m_Errors = Value
            End Set
        End Property
        Private m_Errors As List(Of ResponseError)

        Public Sub New()
            Errors = New List(Of ResponseError)()
        End Sub
    End Class
End Namespace

File: PayJunction C#/ConsoleApplication2/PayJunction/Response.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PayJunction
{
    abstract class Response
    {
        public List<ResponseError> Errors { get; set; }

        public Response()
        {
            Errors = new List<ResponseError>();
        }
    }
}
